code/modules/uplink/uplink_items/nukeops.dm

```diff
@@ -540,6 +540,13 @@
 	cost = 18
 	purchasable_from = UPLINK_NUKE_OPS
 
+/datum/uplink_item/bundles_tc/cyborg_support
+	name = "Cyborg support Kit bag"
+	desc = "A duffle bag containing a full toolbelt including a inducer an spare battery, with a emergency reboot module and a emagg."
```

Comment:
```suggestion
	desc = "A duffle bag containing a full toolbelt (inducer included), an extra bluespace cell, a reboot module, and a cryptographic sequencer, just in case your silicons get turned against you."
```

Reply:
Come on man... these grannars... what the heck...